### DISCUSSION TITLE
Notes aren't taking a template

### DISCUSSION THREAD

**Discussion by djdrysdale** [binyamin/eleventy-garden#46] [Q&A] [answered]
Hello, 

I am new to static sites and have been playing with 11ty Garden as a way to host my digital notes. Things are going well except that I can't seem to get get my notes to take a template. The links work fine, but they look like unstyled text. I created the notes in Obsidian. 

Here are [the notes](https://nostalgic-allen-d6028c.netlify.app/notes/). I'm sure I've done something obvious but I can't see it.

**Comment by binyamin** [comment]
@djdrysdale Sorry that it's not working. What's the link to your code? That would help me debug it.

**Reply by djdrysdale** [reply]
Thanks @binyamin. Here is [my repo](https://github.com/djdrysdale/digital-garden). I don't imagine it's a bug but probably something I've done.

**Reply by djdrysdale** [reply, marked as answer]
Welp not sure how, but I managed to get it going again.
